server/app/controllers/fusor/api/v2/deployments_controller.rb

```diff
@@ -87,5 +89,42 @@ def find_deployment
     def customer_portal_credentials
       { :username => session[:portal_username], :password => session[:portal_password] }
     end
+
+    def save_deployment_attributes
+      path = ::Fusor.log_file_dir(@deployment.label, @deployment.id)
+      if !File.directory?(path)
+        Utils::Fusor::CommandUtils.run_command("mkdir -p #{path}")
+      end
+
+      dep_file = "#{path}/deployment.json"
+      if File.file?(dep_file)
+        Utils::Fusor::CommandUtils.run_command("rm -f #{dep_file}")
+      end
```

Comment:
Hrm I see we're making a json file not a log file. So I understand why there is no log file.